YgoProPatcher/Client.Designer.cs

```diff
@@ -342,55 +342,55 @@ private void InitializeComponent()
             0,
             0});
             this.UpdateCheckerTimeNumeric.ValueChanged += new System.EventHandler(this.UpdateCheckerTimeNumeric_ValueChanged);
-            // 
+            //
             // label1
-            // 
+            //
             this.label1.AutoSize = true;
             this.label1.Font = new System.Drawing.Font("Microsoft Sans Serif", 9F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(238)));
             this.label1.Location = new System.Drawing.Point(537, 227);
             this.label1.Name = "label1";
             this.label1.Size = new System.Drawing.Size(175, 18);
             this.label1.TabIndex = 26;
             this.label1.Text = "Minutes between checks:";
-            // 
+            //
             // UpdateWhenLabel
-            // 
+            //
             this.UpdateWhenLabel.AutoSize = true;
             this.UpdateWhenLabel.Location = new System.Drawing.Point(537, 261);
             this.UpdateWhenLabel.Name = "UpdateWhenLabel";
             this.UpdateWhenLabel.Size = new System.Drawing.Size(0, 17);
             this.UpdateWhenLabel.TabIndex = 27;
-            // 
+            //
             // MimimizeButton
-            // 
+            //
             this.MimimizeButton.Location = new System.Drawing.Point(631, 302);
             this.MimimizeButton.Name = "MimimizeButton";
             this.MimimizeButton.Size = new System.Drawing.Size(150, 31);
             this.MimimizeButton.TabIndex = 28;
-            this.MimimizeButton.Text = "Minimize to TaskBar";
+            this.MimimizeButton.Text = "Minimize to taskbar";
             this.MimimizeButton.UseVisualStyleBackColor = true;
             this.MimimizeButton.Click += new System.EventHandler(this.MinimizeButton_Click);
-            // 
+            //
             // notifyIcon1
-            // 
+            //
             this.notifyIcon1.BalloonTipIcon = System.Windows.Forms.ToolTipIcon.Info;
             this.notifyIcon1.Text = "YgoProPatcher ";
             this.notifyIcon1.Visible = true;
             this.notifyIcon1.MouseClick += new System.Windows.Forms.MouseEventHandler(this.notifyIcon1_MouseClick);
-            // 
+            //
             // StartMinimizedCheckbox
-            // 
+            //
             this.StartMinimizedCheckbox.AutoSize = true;
             this.StartMinimizedCheckbox.Font = new System.Drawing.Font("Microsoft Sans Serif", 10F);
             this.StartMinimizedCheckbox.Location = new System.Drawing.Point(595, 348);
             this.StartMinimizedCheckbox.Name = "StartMinimizedCheckbox";
             this.StartMinimizedCheckbox.Size = new System.Drawing.Size(186, 44);
             this.StartMinimizedCheckbox.TabIndex = 29;
-            this.StartMinimizedCheckbox.Text = "Start minimized and\r\nchecking for updates";
+            this.StartMinimizedCheckbox.Text = "Start minimized and\r\ncheck for updates";
```

Comment:
I’m not sure about this, becase it will keep checking, not check just once.

Reply:
Even so, the way you put it isn't grammatically correct.